src/deadline/client/cli/mcp_server.py

```diff
@@ -0,0 +1,50 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+The `deadline mcp-server` command.
+"""
+
+import sys
+import click
+
+from ._common import _handle_error
+from deadline.client.api._telemetry import get_deadline_cloud_library_telemetry_client
+
+
+@click.command(name="mcp-server")
```

Comment:
When I cloned your branch, I found the `mcp-server` command wasn't available. I had to make the following edit:

```suggestion
from ._main import main


@main.command(name="mcp-server")
```

Reply:
Fixed